### PR TITLE
增加缓存标识

### PR DESCRIPTION
**Pull request by zhanghangt**

### CODE REVIEW

**Comment by hsluoyz** [comment]
@techoner please review.

File: src/adapter/DatabaseAdapter.php [hunk 1]
```diff
@@ -57,7 +57,7 @@ public function savePolicyLine($ptype, array $rule)
      */
     public function loadPolicy(Model $model): void
     {
-        $rows = $this->model->cache('tauthz')->select()->toArray();
+        $rows = $this->model->cache(config('tauthz.enforcers.basic.database.cahce_key', 'tauthz'))->select()->toArray();
```
Comment (leeqvip) [inline src/adapter/DatabaseAdapter.php:60]: If there are multiple enforcers, you need to distinguish the configuration of which enforcer to read, please add it to TauthzService.

**Review by leeqvip — CHANGES_REQUESTED** [review]

File: src/adapter/DatabaseAdapter.php (hunk 1 above)
Comment (zhanghangt) [reply src/adapter/DatabaseAdapter.php:60]: If there are multiple enforcers,You can set the cache's prefix of the thinkphp

Comment (leeqvip) [reply src/adapter/DatabaseAdapter.php:60]: Each group of configuration corresponds to an enforcer, the default is the `basic` group, but you cannot fix it as `basic`.
How to read the configuration by group, please refer to: https://github.com/php-casbin/think-authz/blob/master/src/TauthzService.php#L31

**Pull request closed without merging**